src/Build.UnitTests/Construction/SolutionProjectGenerator_Tests.cs

```diff
@@ -1601,7 +1601,7 @@ public void VenusSolutionDefaultTargetFrameworkVersion()
 
             // v4.0 by default
             ProjectInstance msbuildProject = CreateVenusSolutionProject();
-            Assert.Equal("v4.0", msbuildProject.GetPropertyValue("TargetFrameworkVersion"));
+            Assert.Equal("v4.8", msbuildProject.GetPropertyValue("TargetFrameworkVersion"));
```

Comment:
This seems bad. We shouldn't be changing anything in the product. Is this affected some other way?